nuxeo-jsf/nuxeo-platform-webapp-base/src/main/java/org/nuxeo/ecm/webapp/helpers/EventNames.java

```diff
@@ -189,6 +189,11 @@
      */
     public static final String FLUSH_EVENT = "flush";
 
+    /**
+     * @since 10.3
+     */
+    public static final String MAIN_TABS_CHANGED = "mainTabsChanged";
```

Comment:
Question already asked in the 9.10 PR: why TABS and not TAB?

Reply:
because it's about a change on the main tabs

Reply:
It's a change in the currently selected tab in the "main tabs" category. So I'd say this should be called "main tab changed". The available main tabs didn't change, just the selected one.

Reply:
I wanted to keep this idea of a change on the "main tabs" category, but ok I'll change ; I want my PR to be merged :)